src/patternfly/components/FormControl/form-control.scss

```diff
@@ -1,119 +1,146 @@
+$pf-c-form-control__select--Coordinates: "M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" !default;
+$pf-c-form-control__select--ViewBox: "320" !default;
 
 .pf-c-form-control {
   // Component
   --pf-c-form-control--FontSize: var(--pf-global--FontSize--md);
   --pf-c-form-control--LineHeight: var(--pf-global--LineHeight--md);
-  --pf-c-form-control--Height: calc(#{var(--pf-global--FontSize--md)} * 2 + 4px);
-  --pf-c-form-control--PaddingTop: calc(#{var(--pf-global--spacer--sm)} - 3px); // calc is required to get the propper height here to match buttons
-  --pf-c-form-control--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-form-control--PaddingBottom: calc(#{var(--pf-global--spacer--sm)} - 3px);
-  --pf-c-form-control--PaddingLeft: var(--pf-global--spacer--sm);
   --pf-c-form-control--Color: var(--pf-global--Color--dark-100);
-  --pf-c-form-control--BorderSize: var(--pf-global--BorderWidth--sm);
-  --pf-c-form-control--BorderColor: var(--pf-global--BorderColor--dark);
-  --pf-c-form-control--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-form-control--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-form-control--BorderTopColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-form-control--BorderRightColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-form-control--BorderBottomColor: var(--pf-global--BorderColor--dark);
+  --pf-c-form-control--BorderLeftColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-form-control--BorderRadius: 0;
   --pf-c-form-control--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
 
+  // padding
+  // Note for top/bottom padding, subtracting the border height so the height of the input is consistent with buttons and dropdowns that use the same spacing but draw their borders with pseudo elements
+  --pf-c-form-control--PaddingTop: calc(var(--pf-global--spacer--form-element) - var(--pf-global--BorderWidth--sm));
+  --pf-c-form-control--PaddingBottom: calc(var(--pf-global--spacer--form-element) - var(--pf-global--BorderWidth--sm));
+  --pf-c-form-control--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-form-control--PaddingLeft: var(--pf-global--spacer--sm);
+
+  // hover
+  --pf-c-form-control--hover--BorderBottomColor: var(--pf-global--primary-color--100);
+
+  // focus
+  --pf-c-form-control--focus--PaddingBottom: calc(var(--pf-c-form-control--PaddingBottom) - calc(var(--pf-c-form-control--focus--BorderBottomWidth) - var(--pf-c-form-control--BorderWidth)));
```

Comment:
I bet you have a good reason for this but do you need to include extra calc here? Couldn't it just be
`calc(var(--pf-c-form-control--PaddingBottom) - var(--pf-c-form-control--BorderWidth));`?

Reply:
yeah in this instance, it would achieve the same thing, but only because `calc(var(--pf-c-form-control--focus--BorderBottomWidth) - var(--pf-c-form-control--BorderWidth))` is the same as `var(--pf-c-form-control--BorderWidth)`. However, if you change the val of `var(--pf-c-form-control--focus--BorderBottomWidth)` to 5px, then that padding-bottom adjustment doesn't work because it doesn't include the width of the bottom border.

Reply:
👍